content/academy/apify_platform/deploying.md

```diff
@@ -12,6 +12,35 @@ This section will discuss how to use your newfound knowledge of the Apify platfo
 
 Because actors are basically just chunks of code running in Docker containers, you're able to **_Actorify_** just about anything!
 
+## Our example project
+
+For this section, we'll be turning this example project into an actor:
```

Comment:
Could we add the steps the user will be taking? I think it's good practice to start a section with some outline so that they immediately know what they will learn. It would also help give a bit of a context to this random code example. It should say something like "...we will take this code that sums numbers and turn it into an actor that accepts several numbers on input and saves the sum as output.."